### PR TITLE
init ios

### PR DESCRIPTION
**Pull request by ivansapr**
Run example on iOS device to generate needed files. 

Add empty "locationStatusStream" FlutterEventChannel in iOS module.

### CODE REVIEW

**Comment by xni06** [comment]
What are we reviewing here? Can you please add a description to the pull request? Thanks.

**Comment by ivansapr** [comment]
@xni06 sorry! Update description!

File: example/ios/Flutter/Flutter.podspec [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#
+# NOTE: This podspec is NOT to be published. It is only used as a local source!
```
Comment (xni06) [inline example/ios/Flutter/Flutter.podspec]: As it says, this file is not supposed to be published

Comment (igaurab) [reply example/ios/Flutter/Flutter.podspec]: Hi, I do not have a ios device to test the app in, please let me know if the PR is ready to be merged. Thanks for the contribution.

Comment (ivansapr) [reply example/ios/Flutter/Flutter.podspec]: @xni06 i deleted this file from PR

**Pull request merged**